libnmstate/netinfo.py

```diff
@@ -91,6 +96,30 @@ def _devinfo_bond(dev, devinfo):
             }
     return {}
 
+def _devinfo_ip(dev, devinfo):
+    ip4info = {}
+    connection = dev.get_active_connection()
+    if connection:
```

Comment:
We can reduce indentation and make it easier to reach by exiting early:

if not act_connection:
    return {'enabled': False}
...